src/krbd.cc

```diff
@@ -222,37 +224,50 @@ static int wait_for_udev_add(struct udev_monitor *mon, const char *pool,
             this_image && strcmp(this_image, image) == 0 &&
             this_snap && strcmp(this_snap, snap) == 0) {
           bus_dev = dev;
-          continue;
+          goto check;
```

Comment:
This conditional block is where I had to resolve conflicts manually, and I'm not super confident the conflict resolution is correct.

Reply:
This looks fine.